tests/installation/module_selection/register_module_desktop.pm

```diff
@@ -0,0 +1,18 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Select Desktop Application module.
```

Comment:
I think this would help to understand someone else, to describe at the level of the test layer what is the name of the screen, because both looks really similar. So could could say that we select/register in that screen and even add Full medium or Online medium, but that is optional. wdyt?